_2017_pages/code_of_conduct.md

```diff
@@ -0,0 +1,189 @@
+    <button type="button" class="btn btn-warning btn-lg">Report a Code of Conduct Incident</button>
+  </a>
+</div>
+
+We value the participation of each member of the PyMC community and want all attendees to
+have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to
+show respect and courtesy to other attendees throughout the conference and at all
+conference events, whether officially sponsored by PyMC or not.
+
+To make clear what is expected, all delegates, speakers, exhibitors and volunteers at
+PyMCon event are required to conform to the NumFOCUS Code of Conduct. Organizers will
+enforce this code throughout the event.
+
+We reproduce here certain portions of the NumFOCUS Code of Conduct that are specifically
+relevant to PyMCon. We encourage all PyMCon participants to review the complete
+Code of Conduct.
+
+## The Shorter Version
+
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that
+harassment and sexist, racist, or exclusionary jokes and language are not appropriate for
+PyMCon.
+
+All communication should be appropriate for a professional audience including people of
+many different backgrounds. Sexual language and imagery is not appropriate.
+
+PyMCon is dedicated to providing a harassment-free event experience for everyone,
+regardless of gender, sexual orientation, gender identity, and expression, disability,
+physical appearance, body size, race, or religion. We do not tolerate harassment of
+participants in any form.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+## Code of Conduct Highlights Specific to PyMCon
+
+We reproduce here certain portions of the NumFOCUS Code of Conduct that are specifically
+relevant to PyMCon. We encourage all PyMCon participants to review the complete Code of
+Conduct.
+
+### How to Report
+
+If you believe someone is violating the Code of Conduct, please report this in a timely
+manner. Code of Conduct violations reduce the value of the community for everyone and we
+take them seriously.
+
+You can make a personal or anonymous report by:
+
+* **By Email**: Send a message to any or all Diversity Chairs, which can be reached at
+  oriol.abril.pla@gmail.com and andorra.alexandre@gmail.com. Event organizers will respond
+  promptly.
+* **On Slack**: Join the [PyMCon Slack](https://pymcon.slack.com/join/shared_invite/zt-fk9efmvb-SWvT01TOnykgzFmLgn1iAw)
+  and contact with the Diversity Chairs, Oriol Abril or Alex Andorra.
+* **Online Form**: A variant of the NumFOCUS Code of Conduct Report is available for
+  PyMCon [here](https://numfocus.typeform.com/to/kMgloi8t) (your name and contact
+  information are not required).
+
+During PyMCon, we recommend contacting via email or Slack so we can take
+immediate action if necessary. In addition to contacting directly with the Diversity
+committee, we also encourages you to make a report using our reporting form.
```

Comment:
In addition to contacting directly ~~with~~ the Diversity committee, we also encourage~~s~~ you to make a report using our reporting form.